metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -39,9 +39,16 @@ object ClientCommands {
        |""".stripMargin
   )
 
+  val RunCode = Command(
+    "metals-scala-run",
+    "run",
+    "runs code"
+  )
+
   def all: List[Command] = List(
     RunDoctor,
     ToggleLogs,
-    FocusDiagnostics
+    FocusDiagnostics,
+    RunCode
```

Comment:
This list is used to auto-generate the documentation on the website. If users are not expected to trigger this command manually then we can leave it out of the list. 

A better name for the variable instead of `all` might be `allUserFacingCommandThatShouldBeDocumentedOnTheWebsite` 😅

Reply:
I will add a comment here :)

on the other hand, users are not expected to trigger this command but must implement it to be able to run/debug code. Where can that be expresses?

Reply:
The contract of code lens is that the client executes this command when the lens is selected so we shouldn't need to document this functionality.